assets/css/v2/on_bus/time.scss

```diff
@@ -0,0 +1,44 @@
+.departure-time {
+  display: inline-block;
+  align-items: center;
+  justify-content: end;
+  width: 100%;
+  margin-top: 25.5px;
+  font-family: Inter;
+  font-size: 47px;
+  font-style: normal;
+  vertical-align: middle;
+}
+
+.departure-times-with-crowding,
+.departure-time-with-crowding {
+  height: 100%;
+}
+
+.departure-crowding {
+  width: 0;
+  height: 0;
```

Comment:
Can we use `display: none` here? That seems more appropriate for what we want to do.